source-map-support.js

```diff
@@ -12,6 +12,8 @@ try {
   /* nop */
 }
 
+var newBuffer = require('buffer-from');
```

Comment:
Would you mind renaming this to `bufferFrom` or similar? ☺️

Reply:
ab0ec53cab3a18f2e9dba71412a02a5a362d9de8